packages/cpsuLambda/src/schema/format_1/transformer.ts

```diff
@@ -99,7 +99,16 @@ function populateTemplate(
     )
     return Err(`Invalid business status on item ${prescriptionItem.itemID}`)
   } else {
-    entry.resource!.businessStatus!.coding![0].code = businessStatus.value()
+    const statusValue = businessStatus.value()
+    entry.resource!.businessStatus!.coding![0].code = statusValue
+
+    if (prescriptionDetails.deliveryType === "Robot Collection") {
```

Comment:
This is already being done on line 114

Reply:
The unnecessary code block has been removed.